argilla/src/argilla/datasets/_export/card/argilla_template.md

```diff
@@ -0,0 +1,185 @@
+---
+# For reference on model card metadata, see the spec: https://github.com/huggingface/hub-docs/blob/main/datasetcard.md?plain=1
+# Doc / guide: https://huggingface.co/docs/hub/datasets-cards
+{{ card_data }}
+---
+
+# Dataset Card for {{ repo_id.split("/")[-1] }}
+
+{% if homepage_url %}
+- **Homepage:** [{{ homepage_url }}]({{ homepage_url }})
+{% endif %}
+{% if repo_url %}
+- **Repository:** [{{ repo_url }}]({{ repo_url }})
+{% endif %}
+{% if paper_url %}
+- **Paper:** [{{ paper_url }}]({{ paper_url }})
+{% endif %}
+{% if leaderboard_url %}
+- **Leaderboard:** [{{ leaderboard_url }}]({{ leaderboard_url }})
+{% endif %}
+{% if point_of_contact %}
+- **Point of Contact:** {{ point_of_contact }}
+{% endif %}
+
+This dataset has been created with [Argilla](https://argilla-io.github.io). As shown in the sections below, this dataset can be loaded into Argilla as explained in [Load with Argilla](#load-with-argilla), or used directly with the `datasets` library in [Load with `datasets`](#load-with-datasets).
```

Comment:
The link doesn't seem to work, should we use the github link as an entry point (there's an overview, link to docs, and we might even get a star?)